src/azure-cli/azure/cli/command_modules/network/_breaking_change.py

```diff
@@ -0,0 +1,13 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from azure.cli.core.breaking_change import register_command_group_deprecate
+
+
+register_command_group_deprecate(
+    "network public-ip ddos-protection-statu",
+    redirect="az network public-ip ddos-protection-status",
```

Comment:
Is the redirect command available in this version? If not, could the redirect message potentially confuse customers?

Reply:
emmm, how to pre-inform the users about the relationship between the 2 command groups then?

Reply:
Could we provide the new command group in this version? We can offer both the deprecated and recommended groups, allowing customers to transition to the new behavior smoothly. During the breaking change window, we can then remove the deprecated group.